src/dataflow/src/server.rs

```diff
@@ -207,8 +207,10 @@ where
     let command_rxs = {
         let mut rx = switchboard.broadcast_rx(BroadcastToken).fanout();
         let command_rxs = Mutex::new((0..threads).map(|_| Some(rx.attach())).collect::<Vec<_>>());
-        tokio::spawn(rx.shuttle()
-                .map_err(|err| panic!("failure shuttling dataflow receiver commands: {}", err)));
+        tokio::spawn(
```

Comment:
very very minor nit: I think this is just a formatting diff that was meant to be part of the last commit (i do this all the time and do not care at all if you don't want to change)

Reply:
Whoops, thanks! Should be fixed now though.